src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -238,12 +246,12 @@ export default App.extend(extend({
     model.destroy();
   },
   addSubscriptions() {
-    Radio.request('ws', 'add', this.comments.models);
+    Radio.request('ws', 'add', [...this.comments.models, ...this.attachments.models]);
   },
   removeSubscriptions() {
     // for when sidebar is closed before comments api request is finished
-    if (!this.comments) return;
+    if (!this.comments || !this.attachments) return;
```

Comment:
This doesn't seem right to me. I don't know if there's a situation where only one of these is `undefined`. Since the `beforeStart()` requests are called together, right?

Is there another way of guarding against the sidebar being closed before `beforeStart()` can finish?

Reply:
I switched it to use `this.isRunning()` 👍